benchmark/FedHPOB/scripts/gcn/cora_prox.yaml

```diff
@@ -0,0 +1,38 @@
+use_gpu: True
+device: 0
+early_stop:
+  patience: 100
+seed: 12345
+federate:
+  mode: standalone
+  make_global_eval: True
+  client_num: 5
+  total_round_num: 500
+  join_in_info: ['num_sample']
+data:
+  root: data/
+  type: cora
+  splitter: 'louvain'
+  batch_size: 1
+model:
+  type: gcn
+  hidden: 64
+  dropout: 0.5
+  out_channels: 7
+  task: node
+criterion:
+  type: CrossEntropyLoss
+train:
+  local_update_steps: 1
+  optimizer:
+    lr: 0.25
+    weight_decay: 0.0005
+trainer:
+  type: nodefullbatch_trainer
+eval:
+  freq: 1
+  metrics: ['acc', 'correct', 'f1']
+  split: ['test', 'val', 'train']
+fedprox:
+  use: True
+  mu: 5.0
```

Comment:
this hyperparameter is critical in this algorithm. I am wondering how we should consider it in our setting, that is to say, to be searched or fixed?

Reply:
`mu` is to be searched in (0.1 1.0 5.0).